content/blog/aws-eks-auto-mode/index.md

```diff
@@ -0,0 +1,2051 @@
+---
+title: "Getting Started with Amazon EKS Auto Mode in Pulumi"
+date: 2024-12-05T22:06:21+01:00
+draft: false
+meta_desc: "Learn how to use Amazon EKS Auto Mode with Pulumi to streamline your Kubernetes cluster management"
+meta_image: meta.png
+authors:
+    - florian-stadler
+tags:
+    - aws
+    - eks
+    - kubernetes
+---
+
+AWS recently announced [Amazon EKS Auto Mode](https://aws.amazon.com/eks/auto-mode/), a significant enhancement to Amazon EKS that streamlines cluster management by automating infrastructure decisions and operations. Today, we are excited to announce that Pulumi fully supports EKS Auto Mode across our AWS provider ecosystem, enabling you to leverage this powerful feature through infrastructure as code.
+
+<!--more-->
+
+## What is EKS Auto Mode?
+
+EKS Auto Mode extends AWS management beyond the Kubernetes control plane to the underlying infrastructure that powers your workloads. It automates critical aspects of cluster operations including:
```

Comment:
I think that despite linking out to the AWS docs, this section could handle a bit more detail on how it automates k8s clusters, and how that makes things easier and faster.

The Pulumi wrapper is really cool, definitely don't mean to knock it - but from the Pulumi side all the magic lies in the fact that yes, you can use Pulumi for this. Hence, it doesn't hurt to do a secondary plug for "why would I want to use Auto Mode anyway".

Reply:
Good point! I added some more details to this section